### PR TITLE
Improve text content on install page

### PR DESCRIPTION
**Pull request by straight-shoota**
**Before:**
grafik

**After:**
grafik

### CODE REVIEW

File: _pages/install.md [hunk 1]
```diff
@@ -13,37 +18,65 @@ link_actions:
...
+- '[![](/assets/icons/build-circle.svg) Nightlies](/install/nightlies)'
+- '[![](/assets/icons/source-branch.svg) Source](/install/from_sources)'
 ---
 ## Linux
 
 Many Linux distribution have Crystal available in their system packages.
-It might be some older version though. Third party package managers can be
-more up to date.
+It might not have the most recent version though.
+Third party package managers can be more up to date.
```
Comment (beta-ziliani) [inline _pages/install.md:28]:
```suggestion
Third party package managers are typically more up to date.
```

File: _pages/install.md [hunk 2]
```diff
@@ -13,37 +18,65 @@ link_actions:
 - '[![](/assets/install/android.svg) #Android](#android)'
 - '[![](/assets/install/docker.svg) #Docker](#docker)'
 - '[![](/assets/install/construction.svg) #Tools](#developer-tools)'
+- '[![](/assets/icons/build-circle.svg) Nightlies](/install/nightlies)'
+- '[![](/assets/icons/source-branch.svg) Source](/install/from_sources)'
```
Comment (beta-ziliani) [inline _pages/install.md:22]: the assets are missing. for the nightly I'd rather use a 🌙 like icon

**Review by beta-ziliani — CHANGES_REQUESTED** [review]

File: _pages/install.md (hunk 2 above)
Comment (straight-shoota) [reply _pages/install.md:22]: I was trying to find a good icon for nightlies, but couldn't come up with anything great.
If you have a concrete icon in mind that fits into the material set, please let me know.

Comment (beta-ziliani) [reply _pages/install.md:22]: Let's leave that one for now and fix it later.

**Review by beta-ziliani — APPROVED** [review]

**Pull request merged**